template.go

```diff
@@ -51,10 +51,17 @@ https://github.com/{{.GithubRepo}}/issues.
 {{- end}}
 
 ### Dependency Changes
-{{range $dep := .Dependencies}}
+{{if .Dependencies}}
+{{- range $dep := .Dependencies}}
 * **{{$dep.Name}}**	{{if $dep.Previous}}{{$dep.Previous}} -> {{$dep.Ref}}{{else}}{{$dep.Ref}} **_new_**{{end}}
 {{- end}}
+{{- else}}
+This release has no dependency changes
```

Comment:
Should we omit the whole "dependency changes" section if there's no previous deps?

Reply:
I was thinking that, but since it is a top level header, I feel like it is better to say it explicitly. Otherwise you might scroll to the bottom looking for it and wonder if you missed it or was excluded

Reply:
To me, no dependency changes is a good feature too ;)

Reply:
Works for me 👍